torch/distributed/launch.py

```
@@ -220,6 +231,18 @@ def main():
               "your application as needed. \n"
               "*****************************************".format(current_env["OMP_NUM_THREADS"]))
 
+    if args.logdir:
+        # Possibly create the directory to write subprocess log output to.
+        if os.path.exists(args.logdir):
+            if not os.path.isdir(args.logdir):
+                print("passed in --logdir must be a relative path to a directory. Ignoring argument.")
+                args.logdir = None
```

We should probably raise an Error here instead of continuing without logging.